Analysis/Tasks/PWGDQ/dileptonEE.cxx

```diff
@@ -320,20 +336,124 @@ struct DileptonEE {
         dileptonList(event, fValues[VarManager::kMass], fValues[VarManager::kPt], fValues[VarManager::kEta], fValues[VarManager::kPhi], -2, filter);
         for (int i = 0; i < fNTrackCuts; ++i) {
           if (filter & (uint8_t(1) << i)) {
-            fHistMan->FillHistClass(Form("PairsBarrelMM_%s", fTrkCutsNameArray->At(i)->GetName()), fValues);
+            fHistMan->FillHistClass(Form("PairsBarrelLSnn_%s", fTrkCutsNameArray->At(i)->GetName()), fValues);
           }
         }
       }
     }
   }
 };
 
+struct EventMixing {
+  OutputObj<THashList> fOutputList{"output"};
+  HistogramManager* fHistMan;
+  float* fValues;
+  // NOTE: The track filter produced by the barrel track selection contain a number of electron cut decisions and one last cut for hadrons used in the
+  //           dilepton - hadron task downstream. So the bit mask is required to select pairs just based on the electron cuts
+  uint8_t fTwoTrackFilterMask = 0;
+  std::vector<TString> fCutNames;
+
+  //Configurable<std::string> fConfigElectronCuts{"cfgElectronCuts", "jpsiPID1", "Comma separated list of barrel track cuts"};
+  Configurable<std::string> fConfigElectronCuts{"cfgElectronCuts", "lmeePID_TPChadrej,lmeePID_TOFrec,lmeePID_TPChadrejTOFrec", "Comma separated list of barrel track cuts"};
+
+  Filter filterEventSelected = aod::reducedevent::isEventSelected == 1;
+  Filter filterTrackSelected = aod::reducedtrack::isBarrelSelected > uint8_t(0);
+
+  void init(o2::framework::InitContext&)
+  {
+    fValues = new float[VarManager::kNVars];
+    VarManager::SetDefaultVarNames();
+    fHistMan = new HistogramManager("analysisHistos", "aa", VarManager::kNVars);
+    fHistMan->SetUseDefaultVariableNames(kTRUE);
+    fHistMan->SetDefaultVarNames(VarManager::fgVariableNames, VarManager::fgVariableUnits);
+
+    TString histNames = "";
+    TString configCutNamesStr = fConfigElectronCuts.value;
+    if (!configCutNamesStr.IsNull()) {
+      std::unique_ptr<TObjArray> objArray(configCutNamesStr.Tokenize(","));
+      for (int icut = 0; icut < objArray->GetEntries(); ++icut) {
+        fCutNames.push_back(objArray->At(icut)->GetName());
+        histNames += Form("PairsBarrelMEULS_%s;PairsBarrelMELSpp_%s;PairsBarrelMELSnn_%s;", fCutNames[icut].Data(), fCutNames[icut].Data(), fCutNames[icut].Data());
+        fTwoTrackFilterMask |= (uint8_t(1) << icut);
```

Comment:
As commented above, the mask is not needed

Reply:
done